Rakefile

```diff
@@ -41,6 +41,11 @@ Dir.glob('tasks/**/*.rake').each do |rakefile|
   safe_load rakefile
 end
 
+def has_docker?
+  system("docker version > /dev/null 2>&1")
+  0 == $?.exitstatus
+end
+
```

Comment:
`docker version` fails, when boot2docker is installed, but not started. In this case rake-compiler-dock will use boot2docker to start the VM on it's own. The `system` method already returns true or false. So you might use:
``` ruby
def has_docker?
  system("docker version > /dev/null 2>&1 || boot2docker version > /dev/null 2>&1")
end
```